pkg/odo/cli/component/create.go

```diff
@@ -71,11 +71,11 @@ const CreateRecommendedCommandName = "create"
 // since the application will always be in the same directory as `.odo`, we will always set this as: ./
 const LocalDirectoryDefaultLocation = "./"
 
-// DevfilePath is the default path of devfile.yaml
-const DevfilePath = "./devfile.yaml"
+// DevfilePath is the path of devfile.yaml, the default path is "./devfile.yaml"
+var DevfilePath = filepath.Join(LocalDirectoryDefaultLocation, "devfile.yaml")
```

Comment:
`devfile.yaml` should still be a const, please move it back to a const and then use `DevfilePath` as a var?